engine/rpcserver.go

```diff
@@ -3072,3 +3071,40 @@ func (s *RPCServer) GetRecentTrades(_ context.Context, r *gctrpc.GetSavedTradesR
 
 	return resp, nil
 }
+
+func checkParams(exchName string, e exchange.IBotExchange, a asset.Item, p currency.Pair) error {
```

Comment:
remove exchName, rename e -> exch and use e.GetName()

Reply:
panics if e is empty